pkg/server/query/entryHandlers.go

```diff
@@ -385,6 +386,17 @@ func postPqsClearHandler() func(ctx *fasthttp.RequestCtx) {
 	}
 }
 
+func postPqsDeleteHandler() func(ctx *fasthttp.RequestCtx) {
+	return func(ctx *fasthttp.RequestCtx) {
+		err := writer.DeletePQSData()
+		if err != nil {
+			utils.SetInternalError(ctx, "Error while deleting PQS data")
+			return
+		}
```

Comment:
you should also log this `err`. One option is to use `utils.SendInternalError()`:
```
		err := writer.DeletePQSData()
		if err != nil {
			utils.SendInternalError(ctx, "Error while deleting PQS data", "", err)
			return
		}
```
See https://github.com/siglens/siglens/blob/develop/pkg/utils/httpserverutils.go#L902-L941

Reply:
The error are already logged in DeletePQSData, do we want duplicate logs ?

Reply:
addressed in the latest commit.